src/esssans/__init__.py

```diff
@@ -9,9 +9,25 @@
 except importlib.metadata.PackageNotFoundError:
     __version__ = "0.0.0"
 
-from . import beam_center_finder, common, conversions, i_of_q, normalization, sans2d
+from . import (
+    beam_center_finder,
+    common,
+    conversions,
+    i_of_q,
+    io,
+    metadata,
+    normalization,
+    sans2d,
+)
 
-providers = conversions.providers + i_of_q.providers + normalization.providers
+providers = (
```

Comment:
I changed providers to tuples everywhere in order to prevent accidental modifications by users. It is tempting to use code like `sans.providers.append`. But this leads to type errors because Sciline wants providers to be passed as a list. IMHO, this should be changed in Sciline, it should accept any sequence.